Backend/starter_project_dotnetforce7/BlogApp2/BlogApp.Identity/Services/AuthService.cs

```diff
@@ -0,0 +1,138 @@
+﻿using BlogApp.Application.Constants;
+using BlogApp.Application.Contracts.Identity;
+using BlogApp.Application.Models.Identity;
+using BlogApp.Identity.Models;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BlogApp.Identity.Services
+{
+    public class AuthService : IAuthService
+    {
+        private readonly UserManager<ApplicationUser> _userManager;
+        private readonly SignInManager<ApplicationUser> _signInManager;
+        private readonly JwtSettings _jwtSettings;
+
+        public AuthService(UserManager<ApplicationUser> userManager,
+            IOptions<JwtSettings> jwtSettings,
+            SignInManager<ApplicationUser> signInManager)
+        {
+            _userManager = userManager;
+            _jwtSettings = jwtSettings.Value;
+            _signInManager = signInManager;
+        }
+
+        public async Task<AuthResponse> Login(AuthRequest request)
+        {
+            var user = await _userManager.FindByEmailAsync(request.Email);
+
+            if (user == null)
+            {
+                throw new Exception($"User with {request.Email} not found.");
```

Comment:
It would be better to use the result response instance here too (either here or in your controller)